BeVolt/App/src/main.c

```diff
@@ -593,7 +593,7 @@ int SPITestmain(){
 // Debug UART Test
 #include "UART.h"
 int main(){
-	UART3_Init(115200);
+	UART1_Init(115200);
```

Comment:
According to @SijWoo, the new master board is using UART3.

Reply:
After this semester, I'm officially deprecating the 1st version board so we won't ever need to use UART1 for the STM version

Reply:
Sounds good to me. We should make sure that we have a second copy (at least) of the master board before deprecation.